server/src/models/paginator.py

```diff
@@ -0,0 +1,40 @@
+import math
+import typing as t
+
+
+def is_numeric(string: str) -> bool:
+    if string.isnumeric():
+        return True
+    else:
+        try:
+            float(string)
+            return True
+        except ValueError:
+            return False
+
+
+class Paginator:
+    page = 1
+    limit = 10
+
+    def __init__(self, count_games: t.Optional[int], page: t.Optional[str], limit: t.Optional[str]):
```

Comment:
There is no optional variables more, when you become to provide default values.
And just `count` or `total_count`, don't attach to some entity